src/image_cache/mod.rs

```diff
@@ -43,18 +43,17 @@ pub use self::errors::Result;
 use self::errors::*;
 
 pub fn get_image_size_estimate(width: u32, height: u32) -> isize {
+	// In an SRGB image, each pixel is 4 bytes.
```

Comment:
SRGB -> RGBA

SRGB: Standard Red Green Blue - This describes the color space and gamma of the image data. (An SRGB image could have rgb channels or rgba. I could even image the term being used for a single channel just to describe the gamma)
RGBA: Red Green Blue Alpha - This describes the channels that are in the image.